modules/web-console/web-console-server/src/main/resources/i18/mongo_migration.properties

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright 2019 GridGain Systems, Inc. and Contributors.
+#
+# Licensed under the GridGain Community Edition License (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+migration.mongo.url.empty.0=MongoDB URL was not specified in configuration.
+migration.mongo.url.empty.1=Migration from old Web Console will not be executed.
+migration.mongo.url.empty.2=Migration instructions: https://www.gridgain.com/docs/web-console/latest/migration
+
+migration.mongo.non.empty.database=Database was already migrated. Consider to disable migration in application settings
+
+migration.mongo.stated=Migration started...
+migration.mongo.finished=Migration finished
+migration.mongo.failed=Migration failed
```

Comment:
Missing empty line in the end of file. Please, configure IDEA IDE.

Reply:
Removed